CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * <How do the end-user experience this issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
 * "find first" on Decimal128 field with value NaN does not find objects ([6182](https://github.com/realm/realm-core/issues/6182), since v6.0.0)
 * Value in List of Mixed would not be updated if new value is Binary and old value is StringData and the values otherwise matches ([#6201](https://github.com/realm/realm-core/issues/6201), since v6.0.0)
+* Fixed diverging history in flexible sync if writes occur during bootstraps ([#5804](https://github.com/realm/realm-core/issues/5804), since v11.7.0)
```

Comment:
Can we be a little more descriptive here about what you'd have to do to hit the bug that's being fixed?